test/TorchSharpTest.WithCudaBinaries/TorchSharpTest.WithCudaBinaries.csproj

```diff
@@ -54,6 +54,13 @@
     <Compile Include="..\..\src\TorchSharp\netstandard.cs" />
   </ItemGroup>
 
+  <ItemGroup>
+      <AssemblyAttribute Include="Xunit.CollectionBehavior">
```

Comment:
Assuming this directive is correct, the tests still fail despite being run sequentially.

Reply:
Did you add it now?

Reply:
If a test fails consistently in the same way, it's not due to concurrency.

Reply:
BTW, I think all of the tests should already be in a collection called 'Sequential'

Reply:
Take a look at the failures in the Windows builds in the latest pipeline instance:

https://dev.azure.com/dotnet/TorchSharp/_build/results?buildId=97107&view=results

They're all random failures, none of them involving your unit test.

Reply:
Maybe it runs two assemblies in parallel: https://xunit.net/docs/running-tests-in-parallel#:~:text=Console%20Runner